tests/robustness/linearizability_test.go

```diff
@@ -197,8 +197,12 @@ func runScenario(ctx context.Context, t *testing.T, lg *zap.Logger, clus *e2e.Et
 	// Run multiple test components (traffic, failpoints, etc) in parallel and use canceling context to propagate stop signal.
 	g := errgroup.Group{}
 	trafficCtx, trafficCancel := context.WithCancel(ctx)
+	failpointCtx, failpointCancel := context.WithCancel(ctx)
```

Comment:
```suggestion
	watchCtx, watchCancel := context.WithCancel(ctx)
```
Even though failpoints will cancel this, the context and cancel impacts watch.

Reply:
Good point, thanks!